cluster/olm/assemble/metadata-common.yaml

```diff
@@ -0,0 +1,241 @@
+spec:
+  customresourcedefinitions:
+    owned:
+      - kind: CephCluster
+        name: cephclusters.ceph.rook.io
+        version: v1
+        displayName: Ceph Cluster
+        description: Represents a Ceph cluster.
+      - kind: CephBlockPool
+        name: cephblockpools.ceph.rook.io
+        version: v1
+        displayName: Ceph Block Pool
+        description: Represents a Ceph Block Pool.
+      - kind: CephFilesystem
+        name: cephfilesystems.ceph.rook.io
+        version: v1
+        displayName: Ceph Filesystem
+        description: Represents a Ceph Filesystem.
+      - kind: CephNFS
+        name: cephnfses.ceph.rook.io
+        version: v1
+        displayName: Ceph NFS
+        description: Represents a Ceph NFS interface.
+      - kind: CephObjectStore
+        name: cephobjectstores.ceph.rook.io
+        version: v1
+        displayName: Ceph Object Store
+        description: Represents a Ceph Object Store.
+      - kind: CephObjectStoreUser
+        name: cephobjectstoreusers.ceph.rook.io
+        version: v1
+        displayName: Ceph Object Store User
+        description: Represents a Ceph Object Store User.
+      - kind: Volume
+        name: volumes.rook.io
+        version: v1alpha2
+        displayName: Ceph Volume
+        description: Represents a Ceph volume.
+  displayName: Rook
+  description: |
+    Rook Storage Orchestration for Kubernetes
```

Comment:
Since each storage provider has its own Rook operator, we will need a separate OLM integration for each storage provider. Thus, we would need to make this specific to Ceph. Each provider would then need to do the work separately to integrate with OLM. So the title and description will need to be specific to Ceph.

Reply:
Perhaps all the files should go under `cluster/olm/ceph`